meta-resin-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -15,6 +15,11 @@ while getopts 'f:i:r' flag; do
 	esac
 done
 
+if [ "$local_image" =  "" ] && [ "$remote_image" = "" ]; then
```

Comment:
Can we fixup this commit?

Reply:
why?

Reply:
`hostapp-update` is introduced with this PR so it would be easier to include all the changes from the beginning. I don't see the advantage of having multiple commits on a file that is introduced here. It gets a little confusing when reviewing by commit.

Reply:
This looks like a fix to another commit in the same PR. Doesn't feel like a new feature. Maybe you missed it when you rebased?

Reply:
I forgot to comment here. I didn't fixup on purpose because it's a clear improvement and doesn't undo something from a previous commit of the same PR. Can we leave it as is?

Reply:
Sure. Sounds good.